istio-operator/istio-operator.spec

```diff
@@ -78,10 +96,20 @@ cd tmp/_output/bin/
     cp -pav stripped/istio-operator $RPM_BUILD_ROOT%{_bindir}/
 %endif
 
+# install the charts
+mkdir -p $RPM_BUILD_ROOT%{_sysconfdir}/istio-operator/%{charts_version}
+cd OPERATOR/src/github.com/maistra/istio-operator/tmp/build/
+cp -rpav _output/helm/ $RPM_BUILD_ROOT%{_sysconfdir}/istio-operator/%{charts_version}
```

Comment:
Is this what you were thinking, and is it correct?

Reply:
Yep. Exactly what I was thinking. Have you tested it? I can pull it and run it if you want.

Reply:
Thanks

Reply:
That would be great if you could test it really quickly.  TBH, I have no idea what I'm doing, nor how to test.  (I'm sure that fills you with confidence. ;))

Reply:
./update.sh

followed by
fedpkg --release el7 srpm

followed by 
copr build @bavery/istio istio-operator-dda6d0aaada2d5bdaf82a6f1ce316877673c829a.tar.gz 

Tested it and it works fine.